{"uuid": "34035426-7649-40ae-9397-c48e2daa0d2c", "children": ["b9653bf9-d35e-4909-affe-52c0839c0933"], "befores": [{"name": "_xunit_setup_class_fixture_Test_Commerce", "status": "passed", "start": 1673597221228, "stop": 1673597226361}], "afters": [{"name": "_xunit_setup_class_fixture_Test_Commerce::0", "status": "broken", "statusDetails": {"message": "AttributeError: type object 'Test_Commerce' has no attribute 'self'\n", "trace": "  File \"/Users/a00/Library/Python/3.8/lib/python/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/Users/a00/Library/Python/3.8/lib/python/site-packages/_pytest/fixtures.py\", line 916, in _teardown_yield_fixture\n    next(it)\n  File \"/Users/a00/Library/Python/3.8/lib/python/site-packages/_pytest/python.py\", line 868, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"/Users/a00/Library/Python/3.8/lib/python/site-packages/_pytest/python.py\", line 786, in _call_with_optional_argument\n    func(arg)\n  File \"/Users/a00/PycharmProjects/pythonProject3/ddt/Commerce_test.py\", line 14, in teardown_class\n    self.self.quit()\n"}, "start": 1673597226384, "stop": 1673597230391}], "start": 1673597221228, "stop": 1673597230391}